tests/suites/test_suite_shax.function

```diff
@@ -102,8 +102,22 @@ void sha384( data_t * src_str, data_t * hash )
 }
 /* END_CASE */
 
+/* BEGIN_CASE depends_on:MBEDTLS_SHA512T256_C */
+void sha512t256( data_t * src_str, data_t * hash )
+{
+    unsigned char output[65];
+
+    memset(output, 0x00, sizeof(output));
+
+
```

Comment:
I see other tests have the same spurious blank line, if you also rename `mbedtls_sha1()` to `sha1()`, maybe you could remove these blank lines.

Reply:
(Note: we use the rocket emoji to indicate "Ack, request changes applied")